### PR TITLE
Feature/create pantry view

### PR DESCRIPTION
**Pull request by Logandv3**
## Description

Please include a summary of the change and/or functionality implementation:
This pull request allows the user to add to pantry and see those changes in their pantry.  It also implements an enable/disable on the cook now button.

## Type of change

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)

## How Has This Been Tested?
Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [x] open index.html
- [x] dev tools
- [x] terminal

## Checklist:

- [x] I have performed a self-review of my own code
- [x] No console error messages

## Additional info
Please include any other relevant information here:
The cook now is not working quite right and not showing updates on the DOM.

### CODE REVIEW

File: src/apiCalls.js [hunk 1]
```diff
@@ -17,18 +19,18 @@ export const userPantry = (pantryUpdate) => {
   fetch('http://localhost:3001/api/v1/users', {
     method: 'POST',
     headers: { 'Content-Type': 'application/json' },
-    body: JSON.stringify(pantryUpdate)
+    body: JSON.stringify(pantryUpdate),
   })
     .then(response => response.json())
-    .then(ingred => console.log(`success: ${ingred}`))
```
Comment (elisebeall) [inline src/apiCalls.js:23]: Writing this particular part of the code for the fetch post process was really helpful for me.  I hope to gain more understanding of the process as we work on our individual projects.

**Review by EricSergeant — APPROVED** [review]
Excellent work getting pantry features in place on the page.  We'll return to the functionality of removing items from pantry when recipe is cooked.

File: src/classes/Pantry.js [hunk 2]
```diff
@@ -27,7 +46,7 @@ class Pantry {
             ingredientID: ingred.id,
             ingredientModification: -Math.abs(ingred.quantity)
           }
-          userPantry(pantryUpdate)
+          userPantry(pantryUpdate);
```
Comment (elisebeall) [inline src/classes/Pantry.js:49]: Nice semi-colon.

File: src/domUpdates.js [hunk 3]
```diff
@@ -105,6 +106,63 @@ let domUpdates = {
     });
 },
 
+  showPantry() {
```
Comment (elisebeall) [inline src/domUpdates.js:109]: Look!  We've got a pantry!

File: src/index.html [hunk 4]
```diff
@@ -49,6 +51,23 @@ <h3 class="list-btns hidden" id="onCookingList">☑︎ On Cooking List</h3>
         </ol>
         <p id="recipeCost">COST: </p>
       </article>
+      <article class="pantry-item hidden" id="pantryView">
```
Comment (elisebeall) [inline src/index.html:54]: I believe this is where the food goes...

File: src/scripts.js [hunk 5]
```diff
@@ -10,14 +10,16 @@ import domUpdates from './domUpdates';
 
 let allRecipes = [];
 let selectedTags = [];
-let currentUser, ingredientsData, recipeData, usersData;
+let currentUser, ingredientsData, recipeData, usersData, pantry, currentRecipe;
```
Comment (elisebeall) [inline src/scripts.js:13]: Oh nooooo, more global variables.  We've failed Turing.

File: src/styles.css [hunk 6]
```diff
@@ -1,125 +1,134 @@
 * {
```
Comment (elisebeall) [inline src/styles.css:1]: We need to revisit this CSS file as it appears something has happened with our indentation!

**Review by elisebeall — COMMENTED** [review]
***We need to revisit the CSS file, as something appears to have gone wrong with the indentation of the entire file!***

**Pull request merged**